services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx

```diff
@@ -151,9 +151,6 @@ describe("Test renderModalOverlayTableBody", () => {
       mockReportContext.report.fieldData.program.length + 1
     );
 
-    // Correct status
-    expect(await findByAltText("complete icon")).toBeVisible();
-
```

Comment:
Should this be removed? Maybe another way to check if this is visible or as a last resort a data-test-id check would work here instead